webui/react/src/components/StateSelectFilter.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import { boolean, text, withKnobs } from '@storybook/addon-knobs';
+import React from 'react';
+
+import StateSelectFilter from './StateSelectFilter';
+
+export default {
+  component: StateSelectFilter,
+  decorators: [ withKnobs ],
+  title: 'StateSelectFilter',
+};
+
+export const Default = (): React.ReactNode => (
+  <StateSelectFilter />
```

Comment:
Thinking the same task as the `UserSelectFilter` of adding `onChange` event to this story, just to test and preserve the state of the selected value (user).